src/main/java/org/isf/medicalstockward/manager/MovWardBrowserManager.java

```diff
@@ -63,7 +63,7 @@ protected void validateMovementWard(MovementWard mov) throws OHDataValidationExc
 		List<OHExceptionMessage> errors = new ArrayList<>();
 		if (description.isEmpty() && mov.isPatient()) {
 			errors.add(new OHExceptionMessage("descriptionPatientEmptyError",
-					MessageBundle.getMessage("angal.medicalstockwardedit.pleaseselectapatient"),
+					MessageBundle.getMessage("angal.common.select.patient.msg"),
```

Comment:
wouldn't be more consistent with the past something like "`angal.common.pleaseselectapatient.msg`" like other similar like "`angal.opd.pleaseselectadisease.msg`"?

Reply:
I was trying to reduce to the length of the keys; it seems to me that "please" in the key is superfluous, redundant, and unneeded.    My preference is to remove the "please" for **all** the keys.

Reply:
the key is the sentence itself, so it can give a exact idea of the value, this is useful for the translators in absence of context so they have not to jump to English back and forth to see the original value.
In other words, according to the key only, someone could translate it as "select patient" only.
Moreover, also in the code would be more clear the message given to the patient without go check the bundles.

> My preference is to remove the "please" for all the keys.

I think these are too many changes to be tracked in one single PR, we agree we can revise in further steps, no worries

Reply:
I will put the "please" back and remove the extra dots in the keys.

Reply:
Thanks @dbmalkovsky !

Reply:
I am doing this because of the dictates of the the project owners but I still believe this is not needed as having worked with translators for 35+ years as they know and understand the context better then one may think.

Reply:
Added the "please", removed the dots, and expanded the key value to better match the actual English message.